handlers/load_test_handler.go

```diff
@@ -92,17 +110,17 @@ func (h *Handler) LoadTestUsingSMPSHandler(w http.ResponseWriter, req *http.Requ
 		loadTestOptions.HTTPQPS = 0
 	}
 
-	// loadGenerator := q.Get("loadGenerator")
+	loadGenerator := testClient.LoadGenerator
 
-	// switch loadGenerator {
-	// case "wrk2":
-	// 	loadTestOptions.LoadGenerator = models.Wrk2LG
-	// default:
-	loadTestOptions.LoadGenerator = models.FortioLG
+	switch loadGenerator {
+	case SMPS.PerformanceTestConfig_Client_wrk2:
+		loadTestOptions.LoadGenerator = models.Wrk2LG
```

Comment:
@kushthedude will this change once we have a Load Generator Interface?

Reply:
I guess so